multi-container/open-policy-agent-sample/README.md

```diff
@@ -0,0 +1,50 @@
+# Running OPA as a side car in Cloud Run
+
+In this tutorial, we will use the [same example](http://openpolicyagent.org/docs/latest/http-api-authorization/) that is documented in the official OPA website, but substitute using [Cloud Run side cars](https://cloud.google.com/run/docs/deploying#sidecars) instead of Docker Compose.
+
+We'll end up with a deployment like this:
+
+![deployment diagram](cloud-run-opa-deployment-diagram.png)
+
+The goal is to use a simple HTTP web server that accepts any HTTP GET request that you issue and echoes the OPA decision back as text. OPA will fetch policy bundles from a simple bundle server. OPA, the bundle server, and the web server will be run as containers in Cloud Run. In a production environment, you likely will have a different bundle server, but it's convenient to package it together here.
+
+# Prerequsites
+
+* An up to date gcloud SDK installed
+* [opa](https://www.openpolicyagent.org/docs/latest/) installed
```

Comment:
```suggestion
* [opa](https://www.openpolicyagent.org/docs/latest/cli/) installed
```

Tweaking url to point directly to the cli page.

Reply:
Thoughts on adding:
* `brew install opa`
* ref a direct link to homebrew site for folks to install opa? https://formulae.brew.sh/formula/opa
* minimum relese version to work the sample?